modules/openthread/Kconfig.features

```diff
@@ -249,6 +249,31 @@ config OPENTHREAD_OTNS
 config OPENTHREAD_PING_SENDER
 	bool "Ping sender support"
 
+config OPENTHREAD_PLATFORM_KEY_REF
```

Comment:
`OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE` is now set in `options.cmake`, so `OPENTHREAD_PLATFORM_KEY_REF` should be used instead of `OPENTHREAD_PLATFORM_KEY_REFERENCES_ENABLE`